Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -117,6 +117,9 @@ $ curl 10.100.28.138:8080
 
 ## Create a Bucket
 
+!!! info
+    This document will guide you to create a bucket with an Object Bucket Claim (OBC). To create a bucket with the experimental COSI Driver, see the [COSI documentation](cosi.md).
```

Comment:
Reword to avoid "you"

```suggestion
    This document is a guide for creating bucket with an Object Bucket Claim (OBC). To create a bucket with the experimental COSI Driver, see the [COSI documentation](cosi.md).
```